pulsar-client-tools-test/src/test/java/org/apache/pulsar/admin/cli/PulsarAdminToolTest.java

```diff
@@ -889,7 +889,7 @@ public void topics() throws Exception {
         verify(mockTopics).revokePermissions("persistent://myprop/clust/ns1/ds1", "admin");
 
         cmdTopics.run(split("list myprop/clust/ns1"));
-        verify(mockTopics).getList("myprop/clust/ns1", null);
+        verify(mockTopics).getList("myprop/clust/ns1", null, null);
```

Comment:
It's better to keep the previous code line as the interface is not removed.

Reply:
we are not removing the code but CLI admin `list` command now uses the new interface which this test is validating.